validator/rpc/structs.go

```diff
@@ -89,3 +89,46 @@ type GetFeeRecipientByPubkeyResponse struct {
 type SetFeeRecipientByPubkeyRequest struct {
 	Ethaddress string `json:"ethaddress"`
 }
+
+// Type of key manager for the wallet, either derivedKeymanagerKind, importedKeymanagerKind, or remote.
+type KeymanagerKind string
+
+const (
+	derivedKeymanagerKind    KeymanagerKind = "DERIVED"
+	importedKeymanagerKind   KeymanagerKind = "IMPORTED"
+	web3signerKeymanagerKind KeymanagerKind = "WEB3SIGNER"
+)
+
+type CreateWalletRequest struct {
+	Keymanager       KeymanagerKind `json:"keymanager"`
+	WalletPassword   string         `json:"wallet_password"`
+	Mnemonic         string         `json:"mnemonic"`
+	NumAccounts      uint64         `json:"num_accounts"`
+	MnemonicLanguage string         `json:"mnemonic_language"`
+}
+
+type CreateWalletResponse struct {
+	Wallet *WalletResponse `json:"wallet"`
+}
+
+type GenerateMnemonicResponse struct {
+	Mnemonic string `json:"mnemonic"`
+}
+
+type WalletResponse struct {
+	WalletPath     string         `json:"wallet_path"`
+	KeymanagerKind KeymanagerKind `json:"keymanager_kind"`
+}
+
+type ValidateKeystoresRequest struct {
+	Keystores         []string `json:"keystores"`
+	KeystoresPassword string   `json:"keystores_password"`
+}
+
+type RecoverWalletRequest struct {
+	Mnemonic         string `json:"mnemonic"`
+	NumAccounts      uint64 `json:"num_accounts"`
+	WalletPassword   string `json:"wallet_password"`
+	Language         string `json:"language"`
+	Mnemonic25ThWord string `json:"mnemonic25th_word"`
```

Comment:
`mnemonic_25th_word` would be better, but is this a breaking change?

Reply:
yeah that would be a breaking change 😓